gometa.go

```diff
@@ -24,6 +24,7 @@ var tmpl = template.Must(template.New("").Parse(`<!DOCTYPE html>
 <html>
 <head>
 <meta name="go-import" content="{{.Host}}{{.Path}} {{.Vcs}} {{.NewURL}}">
+<meta name="go-source" content="{{.Host}}{{.Path}} _ {{.NewURL}}/tree/v2{/dir} {{.NewURL}}/blob/v2{/dir}/{file}#L{line}">
```

Comment:
why "v2"? couldn't it also be "master" or any other tag?
This feels too hardcoded to be workable on all various code platforms.

Any documentation you used to make that decision? Maybe I'm just missing some information.

Also does the #L syntax work on all code sites? Also where does the line come from?

Reply:
I used the examples from https://github.com/golang/gddo/wiki/Source-Code-Links
Couldn't find any other documentation about it.

Reply:
The example uses a specific version tag instead of master. That's a special case in my view.
Also it seems too specific on Github. Will check, if I can find a bit more info on it.

Reply:
Couldn't find anything either. Will keep looking. Happy for you to take another look. It should support the repositories that go-import supports and probably reference master or a specific version set in our URL?

Reply:
Searched a lot but couldn't find any documentation about it. need to check the code itself.

Reply:
Yeah seems like quite a non documented feature. Was looking into supporting domains, but couldn't fine much either.